app/manage-breast-screening/2025/12/exit-pause-resume-appointment/index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Exit, pause and resume an appointment
+description: We're creating a way for mammographers to exit, pause and resume a breast screening appointment when it can't continue as planned.
+date: 2025-12-23
+author: Rebecca Cottrell
+opengraphImage:
+  src: /manage-breast-screening/2025/12/exit-pause-resume-appointment/pause-appointment.png
+  alt: Pause appointment page state. An important banner says, ‘This appointment is paused’. There is a button to ‘Resume appointment’ and a link for ‘Appointment cannot proceed’.
+tags:
+  - beta
+  - prototype
+  - screening
+---
+
+We've added the functionality for users to exit, pause and resume a breast screening appointment. This flow is accessed from a link in the top header ('Exit appointment'). 
+
+We've added the link to the top left because that's a strongly established software pattern: closing a window is normally in the top left, where it’s easy to find. This is normally where the site navigation sits, which we hide during the workflow to allow the user to focus on the task. 
```

Comment:
Link to workflow blog post? https://design-history.prevention-services.nhs.uk/manage-breast-screening/2025/11/a-new-appointment-workflow/